test/validators/transfer.ts

```diff
@@ -26,7 +27,7 @@ describe('Transfer Validator Tests', () => {
     clusters.three = helpers.DataGenerator.cluster.new();
   });
 
-  it('Transfer validator emits ValidatorTransferred event', async () => {
+  it('Transfer validator emits "ValidatorTransferred"', async () => {
```

Comment:
Change the 2 below
Transfer validator into a new cluster A-Z
First one: Transfer validator from pod with one validator into an empty pod 
Second one: Transfer validator from pod with two validators into an empty pod